src/runtime_src/xdp/profile/database/static_info_database.cpp

```diff
@@ -115,6 +115,56 @@ namespace xdp {
     aieApplication = true;
   }
 
+  AppFlowType VPStaticDatabase::getAppFlowType() const
+  {
+    return appFlowType;
+  }
+
+  void VPStaticDatabase::setAppFlowType(AppFlowType flowType)
+  {
+    std::lock_guard<std::mutex> lock(appFlowTypeLock);
+    appFlowType = flowType;
+  }
+
+  bool VPStaticDatabase::continueXDPConfig(bool hw_context_flow)
+  {
+    auto flowType = getAppFlowType();
+    switch(flowType) {
+      case AppFlowType::FLOW_TYPE_NOT_SET:
+      {
+        if (hw_context_flow) {
+          setAppFlowType(AppFlowType::REGISTER_XCLBIN_FLOW);
+        } else {
+          setAppFlowType(AppFlowType::LOAD_XCLBIN_FLOW);
+        }
```

Comment:
```suggestion
        if (hw_context_flow)
          setAppFlowType(AppFlowType::REGISTER_XCLBIN_FLOW);
        else
          setAppFlowType(AppFlowType::LOAD_XCLBIN_FLOW);
```
could you remove { } around single line if-else block ?